README.md

```diff
@@ -239,6 +240,92 @@ This already goes with how CSP is currently enforcing its rules canonically in t
 
 `X-frames` and `frame-src` allow controlling what is allowed to be loaded into an `iframe`. Regardless of their use to deny loading content in iframes, ways to crete new same origin realms remain available, such as creating an `about:blank` iframe, opening a new tab using `open` API and more.
 
+## Consideration
+
+Some important points to consider
+
+### Privacy
+
+There isn't much to take into consideration in regards with privacy.
```

Comment:
"This feature doesn't provide any information about the user, and hence doesn't have privacy aspects to consider"?

Reply:
bff2e15b560299110d936035112bc291d4829873